examples/ios/HelloWorldSwift/BUILD

```diff
@@ -91,13 +90,14 @@ ios_ui_test(
 )
 
 # Example using docc_archive to generate documentation
-docc_archive(
-    name = "HelloWorldSwift.doccarchive",
-    default_code_listing_language = "en",
-    dep = ":HelloWorldSwift",
-    diagnostic_level = "information",
-    enable_inherited_docs = True,
-    fallback_bundle_identifier = "com.example.hello-world-swift",
-    fallback_bundle_version = "1.0.0",
-    fallback_display_name = "HelloWorldSwift",
-)
+# TODO: enable once new rules_swift 2.0 symbol graph rule is supported
+# docc_archive(
```

Comment:
Is this a breaking change? Will we need to bump our `compatibility_level`?

Reply:
We shouldn't release rules_apple until: https://github.com/bazelbuild/rules_apple/pull/2487 which should **not** be a breaking change as none of the public API changed.

Commented it out here so we can merge and follow up independently

Reply:
Since CI is already broken because of the app intents, do we want to leave these tests enabled, but failing?

Reply:
hmm yeah and then just manually merge? that sounds good to me, let me drop that comment commit then

Reply:
updated